email/service_ip.py

```diff
@@ -1,12 +1,16 @@
 import socket
 import json
+import os
 
 path_to = './shared'
 
 def save_ip_for_service(service_name):
     HOST_NAME = socket.gethostname()
     SERVICE_IP = socket.gethostbyname(HOST_NAME)
-
+    
```

Comment:
good, +

Reply:
yyyyy nawet nie wiem jak to skomentować, no cóż mam poradzić